src/sdp/DefaultSDP.ts

```diff
@@ -149,4 +214,105 @@ export default class DefaultSDP implements SDP {
 
     return new DefaultSDP(originalSdp);
   }
+
+  withPlanBSimulcast(videoSimulcastLayerCount: number): DefaultSDP {
```

Comment:
Simply revert some changes made in https://github.com/aws/amazon-chime-sdk-js/commit/abc3bf7b27a95152910d2faf8cbbb0a51a42f731#diff-56abeb90f57cd6acbb5beaf736d6ae2f

Reply:
Side note: in that change I also notice we have some tests that still consider Opera to be an unsupported browser:

```
    it('can be constructed with an unsupported browser', () => {
      const mockBuilder = new DOMMockBuilder();
      setUserAgent(SAFARI_USERAGENT);
      setUserAgent(OPERA_USERAGENT);
```

We should use something that will always be unsupported like IE.